packages/shared/src/components/modals/CompanionModal.tsx

```diff
@@ -41,110 +42,117 @@ export default function CompanionModal({
       kind={Modal.Kind.FlexibleCenter}
       size={size}
     >
-      <Modal.Body className="p-0">
```

Comment:
For some reason when running this locally the `p-0` is respected, but when running in production it is ignored? I removed it entirely for now as it is not needed, hopefully this should fix the scroll/padding issue.

Reply:
It could be the tailwind order on production, but ideally should match locally.
Have you run pnpm install latelty? (After the tailwind upgrade)

Reply:
I'm pretty sure I was up to date with `main`, installed the latest as well